modules/team_planner/spec/support/pages/team_planner.rb

```diff
@@ -176,5 +176,17 @@ def expect_wp_not_resizable(work_package)
     def expect_wp_not_draggable(work_package)
       expect(page).to have_selector('.fc-event:not(.fc-event-draggable)', text: work_package.subject)
     end
+
+    def expect_menu_item(name)
+      expect(page).to have_selector('.op-sidemenu--item-title', text: name)
+    end
+
+    def click_menu_item(name)
+      page.find('.op-sidemenu--item-title', text: name).click
+    end
+
+    def expect_no_menu_item(name)
+      expect(page).to_not have_selector('.op-sidemenu--item-title', text: name)
+    end
```

Comment:
These methods already exist in the queryMenu component so you might consider using them instead.